_docs/reference/release-notes.md

```diff
@@ -1,13 +1,84 @@
+
+- **Improved Mixer Latency**. Mixer now clearly delineates configuration-time vs. request-time computations, which avoids doing extra setup work at 
+request-time for initial requests and thus delivers a smoother average latency. Better resource caching also contributes to better end-to-end perf.
+
+- **Reduced Latency for Egress Traffic**. We now forward traffic to external services directly from the sidecar.
+
+#### Traffic Management
+
+- **Egress Rules**. It’s now possible to specify routing rules for egress traffic.
+
+- **New Protocols**. Mesh-wide support for WebSocket connections, MongoDB proxying.
+
+- **Other Improvements**. Ingress properly supports gRPC services, interfacing headless services, better support for health checks, and
+Jaeger tracing.
+
+#### Policy Enforcement & Telemetry
+
+- **Ingress Policies**. Policies can now be applied to north-south traffic in addition to east-west traffic supported in 0.1.
+
+- **Support for TCP Services**. In addition to the HTTP-level policy controls available in 0.1, 0.2 introduces support for TCP-centric policy controls for 
+TCP services.
+
+- **New Mixer API**. The API that Envoy uses to interact with Mixer has been completely redesigned for increased robustness, flexibility, and to support 
+rich proxy-side caching and batching for increased performance.
+
+- **New Mixer Adapter Model**. A new adapter composition model makes it easier to extend Mixer by adding whole new classes of adapters via templates. This 
+new model will serve as the foundational building block for many features in the future.
+
+- **Improved Mixer Build Model**. It’s now easier to build a Mixer binary that includes custom adapters.
+
+- **Mixer Adapter Updates**. The built-in adapters have all been rewritten to fit into the new adapter model. The stackdriver adapter has been added for this
+release. The experimental redisquota adapter has been removed in the 0.2 release, but is expected to come back in production quality for the 0.3 release.
+
+- **Mixer Call Tracing**. Calls between Envoy and Mixer can now be traced and analyzed in the Zipkin dashboard.
+
+#### Security
+
+- **Mutual TLS for TCP Traffic**. In addition to HTTP traffic, mutual TLS is now supported for TCP traffic as well.
+
+- **Identity Provisioning for non-Kubernetes Nodes**. Auth supports a new mechanism using a per-node agent for identity provisioning. This agent runs on each
+node (VM / physical machine) and is responsible for generating and sending out the CSR (Certificate Signing Request) to get certificates from Istio CA. This is an early version of this feature which has limitations (such as requiring a flat network across containers and VMs).
+
+- **Bring Your Own Root Certificates**. Allows users to provide their own key and cert for Istio CA. In addition to that, CA now supports persistent storage 
+using Kubernetes secrets for signing key/cert.
+
 ## Istio 0.1
```

Comment:
Should we split the 0.1 part off into its own file? At some point this will get too long and we should figure out how we want to manage that.